google_apis/google_services_manager.py

```diff
@@ -50,13 +50,13 @@ def download_file_from_drive(self, drive_file_id, return_df=True, remove=False):
         # Download file from google drive with specified file id
         try:
             file = self.drive_client.CreateFile({'id': drive_file_id})
-            file_name = '{}.csv'.format(file['title'])
+            file_name = file['title']
```

Comment:
this doesnt break anything right

Reply:
Nope. The new files are named “GroupA.csv”, etc. so we don’t have to add the extension unlike before. The files are deleted afterwards anyways